org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/XMLSchemaDiagnosticsTest.java

```diff
@@ -509,6 +509,18 @@ public void cvc_datatype_valid_1_2_3Empty() throws Exception {
 				d(1, 1, 1, 10, XMLSchemaErrorCode.cvc_type_3_1_3));
 	}
 
+	@Test

```

Comment:
Would be helpful to link the original issue for this test case

Reply:
fixed